docs/en/query_language/functions/other_functions.md

```diff
@@ -785,18 +785,48 @@ SELECT throwIf(number = 3, 'Too many') FROM numbers(10);
 Code: 395. DB::Exception: Received from localhost:9000. DB::Exception: Too many.
 ```
 
-## identity()
+## identity {#identity}
+
+Used for debugging and testing, allows to "break" access by index, and get the result and query performance for a full scan.
+
+**Syntax**
+
+```sql
+identity()
+```
+
+**Returned value**
 
 Returns the same value that was used as its argument. 
 
+**Example**
+
+Query:
+
 ```sql
-SELECT identity(42)
+SELECT identity(43)
 ```
+
+Result:
+
 ```text
-┌─identity(42)─┐
-│           42 │
+┌─identity(43)─┐
+│           43 │
 └──────────────┘
 ```
-Used for debugging and testing, allows to "break" access by index, and get the result and query performance for a full scan.
+
+Query:
+
+```sql
+SELECT (SELECT toDate('2015-01-02'), 'Hello') AS x, x, identity((SELECT 1)), identity((SELECT 1) AS y);
```

Comment:
it's not clear what this example is supposed to show, we'd better use some example dataset to show that query that used to read not so many rows does a full scan if we add identity call